src/openrct2/scripting/ScRide.hpp

```diff
@@ -56,6 +211,24 @@ namespace OpenRCT2::Scripting
             return ride != nullptr ? ride->type : 0;
         }
 
+        std::string classification_get() const
```

Comment:
Small oopsie here I discovered when playing with my plugin. While this function works just fine...